crates/backend/src/codegen.rs

```diff
@@ -173,6 +173,7 @@ impl ToTokens for ast::Struct {
         (quote! {
             #[automatically_derived]
             impl #wasm_bindgen::describe::WasmDescribe for #name {
+                // #[coverage(off)]
```

Comment:
I think we should leave a more descriptive comment instead.

Reply:
After looking through the rest of your code I guess this is just your TODO.

Reply:
I left it in there for now in case you have any suggestions where else it might be worth checking if it could be added. I'm pretty sure that the issue is that I'm missing another `#[coverage(off)]` somewhere but I don't know the codegen code well enough.

I'll write up a separate issue that goes into a bit more depth on the topic and probably just remove these comments. No point having a commented line with the caption "I think  this solves issue X but it doesn't".